tests/cases/fourslash/completionEntryClassMembersWithInferredFunctionReturnType1.ts

```diff
@@ -0,0 +1,31 @@
+///<reference path="fourslash.ts" />
```

Comment:
Oddly, this test does not fail when the fix is reverted when main is merged into this PR. Was this fixed by some other PR?